app/errors/validation_exception.rb

```diff
@@ -0,0 +1,22 @@
+class ValidationException < StandardError
+  attr_accessor :error_messages
+
+  def initialize(error_messages)
+    @error_messages = error_messages
+  end
+
+  def as_json
+    errors = error_messages.map do |error_message|
+      {
+        error: 'ValidationError',
+        message: error_message,
+      }
+    end
+
+    { errors: errors }
+  end
```

Comment:
Enables easy convertion of the errors in the format required by the VendorAPI.
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/controllers/vendor_api/vendor_api_controller.rb#L35